bigtable/benchmarks/endurance_benchmark.cc

```diff
@@ -150,39 +150,30 @@ long RunBenchmark(bigtable::benchmarks::Benchmark& benchmark,
   // We pre-allocate the buffer based on largely a guess, at 2.5ms per call, a
   // run of 30 minutes produces 400 * 15 * 60 elements, allocate twice as much
   // because why not?
-  partial.operations.reserve(kPartialResultsPeriod * 60 * 400);
+  partial.operations.resize(kPartialResultsPeriod * 60 * 400);
```

Comment:
Are you sure this is what you want?  Because `resize()` grows the container, and the `emplace_back()` calls will grow it even more, so your container will have `kPartialResultsPeriod * 60 * 400` empty elements at the beginning.

Reply:
Yes, here resize not required

Reply:
Also I am working on to cleanup this PR